WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -253,6 +255,40 @@ public void onResume() {
     }
 
     @Override
+    public void onStart() {
+        super.onStart();
+        EventBus.getDefault().register(this);
+    }
+
+    @Override
+    public void onStop() {
+        super.onStop();
+        EventBus.getDefault().unregister(this);
+    }
+
+    @SuppressWarnings("unused")
+    public void onEvent(ReaderEvents.FollowedTagsChanged event) {
```

Comment:
I think we need to use `onEventMainThread` instead of `onEvent`, the call to `updateCurrentTag()` will change UI stuff (`setText` here: https://github.com/wordpress-mobile/WordPress-Android/blob/issue/2331-reader-event-bus/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java#L1060-1060) (refreshTags is asynchronous, so that should not induce any problem). I've checked other `onEvent` methods, they look good.